doc/source/conf.py

```diff
@@ -26,7 +23,7 @@
     import fabio
     project_dir = os.path.abspath(os.path.join(__file__, "..", "..", ".."))
     build_dir = os.path.abspath(fabio.__file__)
-    if on_rtd:
+    if os.environ.get('READTHEDOCS') == 'True':
         print("On Read The Docs")
         print("build_dir", build_dir)
         print("project_dir", project_dir)
```

Comment:
Is the whole try/except block needed.
One need to make sure to install the same version of `fabio` as the one that is used to build the doc, but is it worth preventing generating the doc from an installed version?